tree:
  ros__parameters:
    sim: True
    controller_ns: "ur5_controller"

